e2e/code-generator/entrypoint.sh

```diff
@@ -12,9 +12,12 @@ fi
 mkdir -p /root/.config/atest
 mkdir -p /var/data
 
-nohup atest server --local-storage '/workspace/test-suites/*.yaml'&
+nohup atest server --local-storage '/workspace/test-suites/*.yaml' &
 sleep 1
 
+curl http://localhost:8080/server.Runner/RunTest -X POST \
```

Comment:
The generated code is not been executed.

Reply:
By the way, why using `/server.Runner/RunTest` instead of `/server.Runner/GenerateCode`

Reply:
Sorry, this was my oversight